NavigateU/NavigateU/ProfileScreen/Views/ProfileViewController.swift

```diff
@@ -49,17 +51,36 @@ final class ProfileViewController: UIViewController, UITableViewDelegate, UITabl
     }
 
 
-
     private func setupBindings() {
-        profileView?.firstName.text = viewModel.firstName
-        profileView?.lastName.text = viewModel.lastName
-        profileView?.userEmail.text = viewModel.email
-        if let profileImage = viewModel.profileImage {
-            self.profileView?.avatarImage.image = profileImage
-        }
-        self.profileView?.tableView.reloadData()
+        viewModel.$firstName
+            .receive(on: DispatchQueue.main )
+            .sink { [weak self] newValue in
+                self?.profileView?.firstName.text = newValue
+            }
+            .store(in: &cancellables)
+        viewModel.$lastName
+            .receive(on: DispatchQueue.main)
+            .sink { [weak self] newValue in
+                self?.profileView?.lastName.text = newValue
+            }
+            .store(in: &cancellables)
+
+        viewModel.$email
+            .receive(on: DispatchQueue.main)
+            .sink { [weak self] newValue in
+                self?.profileView?.userEmail.text = newValue
+            }
+            .store(in: &cancellables)
+        viewModel.$profileImage
+            .receive(on: DispatchQueue.main)
+            .sink { [weak self] newImage in
+                self?.profileView?.avatarImage.image = newImage
+                self?.profileView?.tableView.reloadData()
+            }
+            .store(in: &cancellables)
```

Comment:
this good fix, but you can subscribe at whole viewModel, and when some property will be changed, you will redraw your screen with all properties (maybe some optimization downgrade, but there is no problem with so small calculations)